ui_framework/components/icon/kui_icon.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+export function KuiIcon({ className }) {
+  const classNames = ['kuiButton__icon', 'kuiIcon', className];
+  return <span aria-hidden="true" className={ classNames.join(' ') }/>;
```

Comment:
Can we add the [classnames](https://www.npmjs.com/package/classnames) dependency and use it here instead of using `join`? It will become more useful when we have more complex logic for adding class names, e.g. setting a special "clickable" class to labels if they're associated with an input via the `for` attribute.

```javascript
import classNames from 'classnames';

const classes = classNames('kuiButton__icon kuiIcon', classname);
```